### PR TITLE
[Backlog-17630] Decoupling of Add Sequence step + [BACKLOG-18430] Separate already decoupled PDI steps and job entries into discrete plugins

### PR DESCRIPTION
**Pull request by cardosov**

### CODE REVIEW

File: plugins/core/addsequence/impls/step/pom.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>pdi-addsequence-impls</artifactId>
+    <groupId>org.pentaho.di.plugins</groupId>
+    <version>8.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>pdi-addsequence-step-core</artifactId>
```
Comment (tmcsantos) [inline plugins/core/addsequence/impls/step/pom.xml]: should be renamed to `pdi-addsequence-step-impl`

File: plugins/core/addsequence/impls/step/pom.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <dependency>
+      <groupId>pentaho-kettle</groupId>
+      <artifactId>kettle-engine</artifactId>
+      <version>${kettle.version}</version>
+      <classifier>tests</classifier>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-module-junit4</artifactId>
```
Comment (tmcsantos) [inline plugins/core/addsequence/impls/step/pom.xml]: this is not used

File: plugins/core/addsequence/impls/step/pom.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-api-mockito</artifactId>
```
Comment (tmcsantos) [inline plugins/core/addsequence/impls/step/pom.xml]: this is not used

**Comment by cardosov** [comment]
New approach on decoupling is to be attempt

**Pull request closed without merging**